app/models/message.rb

```diff
@@ -214,6 +216,14 @@ def save_story_info(story_info)
 
   private
 
+  def ensure_processed_message_content
```

Comment:
could use dig here to prevent all the specs from failing

```suggestion
if inbox.channel_type == 'Channel::Email'
  text_content_quoted = content_attributes.dig(:email, :text_content, :quoted)
  html_content_quoted = content_attributes.dig(:email, :html_content, :quoted)

  text_content_quoted || html_content_quoted || content
end
```